docs/reference/release-notes/v2.29/index.md

```diff
@@ -0,0 +1,133 @@
+
+If you want to turn it on/off, you can set the corresponding flag inside `gatsby-config.js`:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  flags: {
+    LAZY_IMAGES: true,
+  },
+  plugins: [], // your plugins stay the same
+}
+```
+
+## Improvements to our CLI
+
+In [v2.27](../v.27/index.md#create-gatsby) we introduced `create-gatsby`, a new and interactive way to create a Gatsby site. You can run it in your terminal with `npm init gatsby`.
+
+A couple of papercuts were fixed but we also added new features:
+
+- Prompt for your site name instead of a folder name
+- Automatically add this name to your `siteMetadata` and `package.json`
+- Add `-y` flag. This flag bypasses all prompts other than naming your site
+
+The regular `gatsby-cli` received a new command to list out all plugins in your site by running `gatsby plugin ls`.
+
+## Experimental: Parallel data sourcing
+
+In [v2.28](../v2.28/index.md#experimental-parallel-data-sourcing) we gave a sneak peak at a new feature that enables parallel data sourcing. As a recap:
+
+> Plugin APIs in Gatsby run serially. Generally this what we want as most API calls are CPU/IO bound so things are fastest letting each plugin have the full undivided attention of your computer. But source plugins are often _network_ bound as they're hitting remote APIs and waiting for responses. We tried [changing the invocation of `sourceNodes` to parallel](https://github.com/gatsbyjs/gatsby/pull/28214) on a few sites with 4+ source plugins and saw a big speedup on sourcing (40%+) as they were no longer waiting on each other to start their API calls.
+
+You're now able to activate this experiment in the stable release of Gatsby by adding a flag to your `gatsby-config.js`:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  flags: {
+    PARALLEL_SOURCING: true,
+  },
+  plugins: [], // your plugins stay the same
+}
+```
+
+Please give us your feedback in the [umbrella discussion](https://github.com/gatsbyjs/gatsby/discussions/28336).
+
+## Performance improvements
+
+We were able to ship a bunch of performance improvements both to Gatsby itself and its plugins:
+
+- The PR [#28375](https://github.com/gatsbyjs/gatsby/pull/28375) fixed an unguided search in `gatsby-source-contentful` that can significantly drop your time for an incremental build (and perhaps others). In this case (for a site with 80k elements) it dropped the time from 5 minutes to sub-second.
+- The PR [#28438](https://github.com/gatsbyjs/gatsby/pull/28438) prevents some redudant calculations resulting in 5% improvements for bigger sites using `gatsby-source-contentful`
+- The PR [#28525](https://github.com/gatsbyjs/gatsby/pull/28525) drops the `async` keyword from the `wrappingResolver` that wraps all resolvers passed on to GraphQL. In practice this means a 5-10% overall improvement to the runtime of a site.
```

Comment:
I think that for release notes this should be more "human readable" for folks without deep knowledge about gatsby / gatsby-source-contentful internals. So instead of "how" perf was improved we should rather say what/when?

```suggestion
### `gatsby`

- Improve query running performance for sites with large amount of data (up to 10% in our tests). See https://github.com/gatsbyjs/gatsby/pull/28525 for details.

### `gatsby-source-contentful`

- Improve performance of data sourcing for large Contentful spaces. See https://github.com/gatsbyjs/gatsby/pull/28375 for details.
- Prevent concurrent requests to Contentful's image processing API. See https://github.com/gatsbyjs/gatsby/pull/28438 for details.
```

/cc @pvdz (in case my understanding of those PRs is wrong / you have better ideas how to word it)